Solutions/GoogleCloudPlatformIAM/Data Connectors/GCP_IAM_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "GCPIAMDataConnector",
+    "title": "Google Cloud Platform IAM (Preview)",
```

Comment:
Can you please remove Preview from title?

Reply:
updated